test_env.py

```diff
@@ -242,7 +242,7 @@ def test_regression_mkl():
 
 def test_conda_channel_priority():
 
-    channel_priority = os.environ.get('CONDA_CHANNEL_PRIORITY', 'False')
+    channel_priority = os.environ.get('CONDA_CHANNEL_PRIORITY', 'disabled')
```

Comment:
This pretty badly fails the travis tests so I'm adding a workaround for now that should be removed in #405